.vscode/launch.json

```diff
@@ -4,6 +4,15 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
+        {
+            "name": "Debug radius cli",
+            "type": "go",
+            "request": "launch",
+            "mode": "debug",
+            "program": "${workspaceFolder}/cmd/rad",
+            "args": "${input:cliArgs}",
+            "console": "integratedTerminal",
+        },
```

Comment:
Did you see the target defined here: https://github.com/radius-project/radius/pull/6905/files#diff-bd5430ee7c51dc892a67b3f2829d1f5b6d223f0fd48b82322cfd45baf9f5e945R64?

Is there a reason why we need two targets that both launch the CLI?

Reply:
Yes I saw the generic launch command. My motivation was the following scenario: 
I am a new contributor and I pick up an iccues that states an error e.g. in the `rad version` command (not super realistic, but as an example). Now I want to debug the command. I found the `version.go` file but now how to easily debug the command without any "detours". 
The added launch command should hopefull yhelp here as it clearly targets the debug scenario. The flow would be:

1. Set a breakpoint where you think it is needed
2. Start the "Debug Radius CLI" configuration
![Screenshot 2023-12-05 081019](https://github.com/radius-project/radius/assets/22294087/bdd8767a-5eb9-4734-bee4-2d916809aecb)

3.  Then you get prompted by VSCode for the command you want to execute (defaulted, to get an idea what to enter and with some additional tooltip) and you enter the command you want to execute
![Screenshot 2023-12-05 081059](https://github.com/radius-project/radius/assets/22294087/60b93ca7-f527-4b2d-94c9-bc62954cdc62)

4. You can start debugging

Does that flow make sense to you? 

I think the other target remains useful as it is more generic, but maybe too generic for contrinbutors starting their journey. Waht do you think?

Reply:
> I think the other target remains useful as it is more generic, but maybe too generic for contrinbutors starting their journey. Waht do you think?

They are both debugging commands. The new one you added prompts for args, so I think it's also useful 👍 

Could you move them to be next to each other in the file name name them similar things. For example:

- Debug rad CLI 
- Debug rad CLI (prompt for args)

Reply:
Can you also add `"preLaunchTask": "Build Radius (all)"` to this? That will help with failure reporting.

The experience for understanding a build error is pretty bad without this.